### PR TITLE
Fix compilation with cython 3

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: scripts/bed_count_by_interval.py [hunk 1]
```diff
@@ -17,7 +17,11 @@
 ranges = {}
 for line in open(bed2):
     fields = line.strip().split()
-    chrom, start, end, = (
+    (
+        chrom,
+        start,
+        end,
+    ) = (
```
Comment (nsoranzo) [inline scripts/bed_count_by_interval.py]: Maybe break this down into 3 separate assignments?

File: scripts/bed_count_overlapping.py [hunk 2]
```diff
@@ -17,7 +17,11 @@
 ranges = {}
 for line in open(bed2):
     fields = line.strip().split()
-    chrom, start, end, = (
+    (
+        chrom,
+        start,
+        end,
+    ) = (
```
Comment (nsoranzo) [inline scripts/bed_count_overlapping.py]: Same.

**Comment by mvdbeek** [comment]
Hmm, the pypy builds are taking a very long time ...

**Comment by nsoranzo** [comment]
> Hmm, the pypy builds are taking a very long time ...

Yep, PyPy v7.3.12 added Python 3.10 and dropped 3.8, so I've just pushed a commit to drop PyPy 3.8 wheels.

**Comment by nsoranzo** [comment]
Thanks @mvdbeek !

Should we make a new release?

**Comment by mvdbeek** [comment]
Yes please! I can (probably) do the release notes and tagging.

**Comment by mvdbeek** [comment]
Nope, doesn't look like I have the permissions to do that.

**Comment by nsoranzo** [comment]
No problem, I am doing it now, it's very lightweight for this repo.

**Comment by nsoranzo** [comment]
New release v0.10.0 is on PyPI.

**Pull request merged**